README.md

```diff
@@ -1,15 +1,48 @@
 # Mink WebDriver Classic Driver
 
+[![PHP Version](http://poser.pugx.org/mink/webdriver-classic-driver/require/php)](https://packagist.org/packages/mink/webdriver-classic-driver)
```

Comment:
Does this work?

Reply:
I noticed that, it does work for other packages, e.g.

http://poser.pugx.org/phpunit/phpunit/require/php -> ![phpunit](http://poser.pugx.org/phpunit/phpunit/require/php)

My guess is that we need to make a packagist release (by adding a git(hub) tag) for it to work the first time.

Reply:
I hope so.

Reply:
This reminds me, @stof could it make sense to tag a new pre-release?

I think the project is in a good enough state for that.